src/payment/strategies/afterpay/afterpay-script-loader.spec.ts

```diff
@@ -65,4 +65,30 @@ describe('AfterpayScriptLoader', () => {
             '//portal.us-sandbox.afterpay.com/afterpay-async.js'
         );
     });
+
+    it('loads widget script for AU & NZ when Feature is enabled', () => {
```

Comment:
since when the feature is enabled the URL does not depend on the country code then you can merge both unit tests (this one and 'loads widget script for US') into the same